### DISCUSSION TITLE
Feature Request: Re-parent Existing Folders

### DISCUSSION THREAD

**Discussion by BaconSkittles10** [androidseb/mapmarker#133] [Q&A] [answered]
I made folders to categories all of my markers into each Hawaiian island. Later, I made markers in Texas and wanted to move the Hawaiian folders into a State of Hawaii folder. I have 150+ markers in Hawaii so I can't go move each marker individually into a new folder with the State of Hawaii parent folder. 

I want a way to move folders to new parents. This could be a drag-and-drop, or a dialogue like the one used to select a folder for a marker. 

Thanks!

**Comment by androidseb** [comment, marked as answer]
Hello, I believe what you're asking for already exists in app:

* Open the "organize" view (3rd button from the top bar)
* Switch to "folder" mode (with the right-most button in the tool bar)
* Select the folder to move
* Click the "move" button
* Select the target folder to move it under

I believe this should address your use case, so I'll close this ticket, but feel free to re-open + comment back with more questions if I missed something, thanks.

**Comment by BaconSkittles10** [comment]
Yep, that is exactly what I was looking for. Thank you!
